UnityProject/Assets/Scripts/Objects/Mining/OreBox.cs

```diff
@@ -26,6 +26,7 @@ private void OnDisable()
 
 	private void AfterMovement(Vector3Int newLocalPosition)
 	{
+		if (isServer == false) return;
```

Comment:
shouldn't you be using `CustomNetworkManager.IsServer`?

Reply:
that's only for non-network behaviours, it's built-in if it's a Network behaviour